error.idgnr.not.supported=[IDGeneration Service] 현재 서비스는 다음 {0} id를 생성하기 위해 지원 하지않습니다.
												

error.idgnr.tableid.notallocate.id=[IDGeneration Service] ID의 블록을 할당 할 수 없습니다. 테이블 이름이 {0}인 행은 {1} 테이블에 존재하지 않습니다.

error.idgnr.null.id=[IDGeneration Service] 많은 시도를 하지만, ID의 블록을 할당 할 수 없습니다.

error.idgnr.get.connection=[IDGeneration Service] 연결을 얻을 수 없습니다. 따라서, ID의 블록을 할당 할 수 없습니다.

error.idgnr.sequenceid.notallocate.id=[IDGeneration Service] ID의 블록을 할당 할 수 없습니다. ID에 대한 쿼리 값을 반환하지 않았습니다.

error.idgnr.greater.maxid=[IDGeneration Service] ID를 제공 할 수 없음. ID가 더 이상 최대 값 {0}에 도달하여 유효합니다.

error.idgnr.invalid.parameter=구성의 주소가 유효한 IP 또는 MAC 주소이어야한다

error.idgnr.select.idblock=[IDGeneration Service] {0} 테이블을 선택하기 예외가 발생했습니다.

error.idgnr.update.idblock=[IDGeneration Service] {0} 테이블을 업데이트하는 예외가 발생했습니다.

warn.idgnr.update.idblock=[IDGeneration Service] {0} 테이블을 업데이트하는 예외가 발생했습니다. 트랜잭션 충돌이 있을 수 있습니다. 다시 시도하십시오.

debug.idgnr.allocate.idblock=[IDGeneration Service] key_table에 대한 {0} ID의 새로운 블록을 할당 {1}.

debug.idgnr.updated.norows=[IDGeneration Service] 업데이트되어 변경된 행이 없습니다..

debug.idgnr.sequenceid.query=[IDGeneration Service] 아이디 사용하여 쿼리를 요청 : {0}

debug.idgnr.init.idblock=[IDGeneration Service] TABLE_NAME = '{0}'에 대한 초기 id 값을 삽입합니다.




